<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Новая запись</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;ФИО&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="FioLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Факультет</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="FacultyComboBox">
       <item>
        <property name="text">
         <string>Авиастроение</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Автоматизация, мехатроника и управление</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Агропромышленный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Безопасность жизнедеятельности и инженерная экология</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Дорожно-транспортный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Инженерно-строительный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Инновационный бизнес и менеджмент</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Информатика и вычислительная техника</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Технология машиностроения</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Кафедра</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="CpecialitiComboBox"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Группа</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="GroupLineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Зачетка</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="BookLineEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Дата рождения</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDateEdit" name="BirthDateEdit"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Телефон</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="PhoneLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="selectRecordButton">
     <property name="text">
      <string>Подтвердить</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="canselButton">
     <property name="text">
      <string>Отмена</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
